### PR TITLE
change usage of variable type to level in clangtidy

### PR DESCRIPTION
**Pull request by risingsmoke**
I have changed the variable type to level in clangtidyissue class. I also have not found any usage of type in the tests.
@La0

### CODE REVIEW

File: src/staticanalysis/bot/static_analysis_bot/clang/tidy.py [hunk 1]
```diff
@@ -261,28 +261,28 @@ def __init__(self, revision, path, line, char, check, message, level='warning'):
         self.notes = []
 
         # TODO: rename all type occurences to level
```
Comment (La0) [inline src/staticanalysis/bot/static_analysis_bot/clang/tidy.py]: Please remove that comment.

File: src/staticanalysis/bot/static_analysis_bot/clang/tidy.py [hunk 2]
```diff
@@ -371,7 +371,7 @@ def as_dict(self):
             'nb_lines': self.nb_lines,
             'char': self.char,
             'check': self.check,
-            'type': self.type,
+            'type': self.level,
```
Comment (La0) [inline src/staticanalysis/bot/static_analysis_bot/clang/tidy.py]: Rename that key to level.

File: src/staticanalysis/bot/static_analysis_bot/clang/tidy.py [hunk 3]
```diff
@@ -261,28 +261,28 @@ def __init__(self, revision, path, line, char, check, message, level='warning'):
...
-        return '[{}] {} {} {}:{}'.format(self.type, self.check, self.path, self.line, self.char)
+        return '[{}] {} {} {}:{}'.format(self.level, self.check, self.path, self.line, self.char)
 
     def build_extra_identifiers(self):
         '''
         Used to compare with same-class issues
         '''
         return {
-            'type': self.type,
+            'type': self.level,
```
Comment (La0) [inline src/staticanalysis/bot/static_analysis_bot/clang/tidy.py]: Rename that key to level.

**Comment by risingsmoke** [comment]
Cool, I will make the necessary changes and update the pull request.
Thanks!

**Comment by La0** [comment]
You have styling issues in your test. Look at the [log here](https://tools.taskcluster.net/groups/FS8HMPDnTY6yWLJGf6Ighg/tasks/KloxhKOSR4muzfQ8VOLAdg/runs/0/logs/public%2Flogs%2Flive.log#L2142)

**Comment by La0** [comment]
Thanks for your work @risingsmoke  !

**Comment by risingsmoke** [comment]
Thanks!

**Comment by sylvestre** [comment]
Bravo!

**Pull request merged**